users/views.py

```diff
@@ -142,6 +148,12 @@ def get(self, request, user_id, token):
                 email_verification_token=token
             )
             
+            if user.is_active:
+                return Response(
+                    {'status': 'success', 'message': 'Email is already verified.'},
+                    status=status.HTTP_200_OK
+                )
+            
```

Comment:
Returning a 200 when the user is already verified is good, but you might want to clarify in the response if further action is possible (e.g., "No email was sent because the address is already verified").